CMakeLists.txt

```diff
@@ -681,6 +681,7 @@ if (BUILD_CLIENT OR BUILD_TTY_CLIENT)
     if (USE_MUMBLE)
         # Mumble link
         add_library(srclibs-mumblelink EXCLUDE_FROM_ALL ${MUMBLELINKLIST})
+        target_link_libraries(srclibs-mumblelink "${LIBS_BASE}")
```

Comment:
Why don't we link to just its dependencies instead of a list of random libraries?